### PR TITLE
refresh example templates including change tag to quickstart

### PR DESCRIPTION
**Pull request by bparees**

### CODE REVIEW

**Comment by bparees** [comment]
@smarterclayton @jwforres ptal.

Note that we're also picking up the new default memory limit parameters that were done for online, so if there are any concerns about those values, now's the time to speak up.

File: examples/quickstarts/cakephp-mysql.json [hunk 1]
```diff
@@ -314,76 +324,102 @@
   ],
   "parameters": [
     {
+      "name": "MEMORY_LIMIT",
+      "displayName": "Memory limit",
```
Comment (spadgett) [inline examples/quickstarts/cakephp-mysql.json]: In the UI, we use headline case for input labels like "Memory Limit". I think the templates should do the same.

https://www.patternfly.org/styles/terminology-and-wording/#capitalization

**Comment by spadgett** [comment]
Should we add a "Quickstart" group to the web console catalog in a follow-on?

File: examples/quickstarts/cakephp-mysql.json [hunk 2]
```diff
@@ -314,76 +324,102 @@
...
+      "description": "Maximum amount of memory the CakePHP container can use",
```
Comment (spadgett) [inline examples/quickstarts/cakephp-mysql.json]: Typically our descriptions end in a period as well.

**Comment by bparees** [comment]
> Should we add a "Quickstart" group to the web console catalog in a follow-on?

no, @jwforres and I discussed this, we're not adding a quickstart group.

**Comment by bparees** [comment]
@gabemontero couple comments from @spadgett about the parameter updates you've been making.  

@spadgett these come from external repos so we need to fix the values there and then reimport, i think i'd like to defer that to a follow up.

**Comment by spadgett** [comment]
> no, @jwforres and I discussed this, we're not adding a quickstart group.

Fair enough. We could add it to the browse dropdown if it makes sense. Then the quickstarts would still be grouped by technology, but you could filter on them if you want.

openshift_web_console

**Comment by bparees** [comment]
@spadgett 

> Fair enough. We could add it to the browse dropdown if it makes sense. 

yes i think @jwforres suggested that as well and i like it.

**Comment by jwforres** [comment]
On Thu, Feb 25, 2016 at 1:45 PM, Ben Parees <notifications@github.com>
wrote:

> @spadgett <https://github.com/spadgett>
>
> Fair enough. We could add it to the browse dropdown if it makes sense.
>
> yes i think @jwforres <https://github.com/jwforres> suggested that as
> well and i like it.
>

Yes

File: examples/quickstarts/cakephp-mysql.json [hunk 3]
```diff
@@ -314,76 +324,102 @@
...
     },
     {
       "name": "DATABASE_ENGINE",
+      "displayName": "Database engine",
       "description": "Database engine: postgresql, mysql or sqlite (default)",
       "value": "mysql"
     },
     {
       "name": "DATABASE_NAME",
+      "displayName": "Database name",
```
Comment (jwforres) [inline examples/quickstarts/cakephp-mysql.json]: also redundant display name and description looks silly, drop the description if there is nothing more detailed to say

**Comment by gabemontero** [comment]
On the template cleanup front, what's our product's official take on "hostname" vs. "host name" and "username" vs. "user name", or with head line case, "Hostname" vs. "Host Name" and "Username" vs. "User Name" ?  Or do we not have an official take (and need to make one)?

I at lease don't see a consensus in general on the web (though perhaps I'm not looking in the right spots).

**Comment by bparees** [comment]
Hostname seems more typical to me.

Username vs User Name i have a less strong opinion on and it seems somewhat context dependent, but Username feels slightly better to me as a reference to a value.

**Comment by spadgett** [comment]
"Username" is recommended by Patternfly for the login screen, so it would make sense here to me as well.

This is a good reference: https://www.patternfly.org/styles/terminology-and-wording/

**Comment by spadgett** [comment]
+1 for Hostname as well. It's what we use in our UI.

openshift_web_console

**Comment by gabemontero** [comment]
thx @bparees @spadgett  ... and yep, i had looked at patternfly before but missed it ... see the "username" bullet now.

**Comment by bparees** [comment]
[merge]

**Comment by bparees** [comment]
I am merging this to get the quickstart/instantapp tag change in.  description changes can be done in a follow on

**Pull request merged**